app/routes/questions/$question.tsx

```diff
@@ -77,35 +77,29 @@ export function Question({
   const title =
     codaStatus && codaStatus !== QuestionStatus.LIVE_ON_SITE ? `WIP - ${codaTitle}` : codaTitle
   const isLoading = useRef(false)
-  const refreshOnToggleAfterLoading = useRef(false)
+
+  const isExpanded = questionState === '_'
+  const isRelated = questionState === 'r'
```

Comment:
I know you're just copying things, but how about using `QuestionState.RELATED` etc.?

Reply:
good idea